topics/Padroes_de_Codigo.md

```diff
@@ -72,18 +72,20 @@ método sobre objetos diferentes.
 
 ## SOLID
 
-SOLID é um acrônimo que engloba os cinco princípios básicos de orientação a
-objetos. Existem variações, mas em geral eles se resumem a:
+SOLID é um acrônimo para os cinco princípios básicos de design de classes
+orientadas a objetos criados por Robert C. Martin (Uncle Bob).
+Princípios dão nomes a conceitos complexos sobre o cdigo,
+mas não são regras ou verdades absolutas.
 
 * *Single responsibility principle* (princípio de responsabilidade única)
   O princípio de responsabilidade única define que uma classe ou método deveria
   existir somente para implementar uma funcionalidade, mas que essa
   responsabilidade seja completamente encapsulada pela entidade.
 * *Open/close principle* (princípio aberto/fechado)
   O princípio aberto/fechado dita que entidades de software deveriam ser abertas
-  à extensão, mas fechadas à modificação. A ideia é que essas entidades
-  (classes, objetos, métodos...) possam ter suas funcionalidades extendidas,
-  mas não seu código modificado.
+  à extensão, mas fechadas à modificação. A ideia é que essas classes
```

Comment:
Sobre essa alteração, se seguirmos o que o próprio Uncle Bob escreveu sobre Open-Closed, o que estás criticando está na realidade correto: 

**"SOFTWARE ENTITIES (CLASSES, MODULES, FUNCTIONS, ETC.)
SHOULD BE OPEN FOR EXTENSION, BUT CLOSED FOR
MODIFICATION."**

[Página do Solid](http://butunclebob.com/ArticleS.UncleBob.PrinciplesOfOod)
[Artigo Open-Closed Principle](https://drive.google.com/file/d/0BwhCYaYDn8EgN2M5MTkwM2EtNWFkZC00ZTI3LWFjZTUtNTFhZGZiYmUzODc1/view)